bookshelf.js

```diff
@@ -0,0 +1,62 @@
+
+
+var button = document.createElement('button');
+button.setAttribute("id", "fetch-books-btn");
+var textNode = document.createTextNode('Fetch Books');
+button.appendChild(textNode);
+//button.innerHTML("Fetch Books");
+button.addEventListener("click",fetchBooks);
+
+var bookDoc = document.getElementsByTagName("body")[0];
+    bookDoc.appendChild(button);
+
+
+function fetchBooks(){
+    const booksJSON = 'https://raw.githubusercontent.com/codeyourfuture/bookshelf-project/master/books.json'
+    fetch(booksJSON)
+        .then(response => response.json())
+        .then(function processBooks(booksJSON){
+            //var bookArray = Array.from(booksJSON);
+            var mainListNode = document.createElement('ul')
+            button.addEventListener("click",inlineFunc);
+            booksJSON.forEach(function(element) {
+                var listNode = document.createElement('li');
+                listNode.setAttribute("id",element.id);
+                var buttonNode = document.createElement("button");
+                var btnTextNode1 = document.createTextNode('⬆');
+                var btnTextNode2 = document.createTextNode('⬇');
+                var textNode = document.createTextNode(element.title + element.author)
+                buttonNode.appendChild(btnTextNode1);
+                buttonNode.appendChild(btnTextNode2);
```

Comment:
You seem to be adding multiple text nodes to the button, but the spec says to add multiple buttons. You'll need to call `document.createElement('button')` twice (and remember to append both to the DOM) to do that.